test/integration/sequelize.test.js

```diff
@@ -567,19 +567,21 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
       expect(result).to.deep.equal(block);
     });
 
-    it('handles alter: true with underscore correctly', async function () {
-      this.sequelize.define('access_metric', {
-        user_id: {
-          type: DataTypes.INTEGER,
-        },
-      }, {
-        underscored: true,
-      });
+    if (dialect !== 'yugabyte'){ // change column type not supported in yugabyte
+      it('handles alter: true with underscore correctly', async function () {
```

Comment:
same, `current.dialect='yugabytedb' ? it.skip : it`
consider for other places also.